src/shared/clients/ssmClient.ts

```diff
@@ -51,10 +59,44 @@ export class SsmClient {
             .flatten()
             .flatten()
             .promise()
-
         return response[0]!
     }
 
+    public async getTargetPlatformName(target: string): Promise<string> {
+        const instanceInformation = await this.describeInstance(target)
+        return instanceInformation.PlatformName!
```

Comment:
style nit (does `prettier` have a setting for this??): 
```suggestion
        const instanceInformation = await this.describeInstance(target)

        return instanceInformation.PlatformName!
```

IMO it's nice to have return statements separated from other statements.